### PR TITLE
resnet_v1 and v2 segmentation bugs from 7e2435e resolved

### PR DESCRIPTION
**Pull request by ahundt**
7e2435e added a squeeze option, which introduced a bug in which logits would never be assigned if squeeze is False. This pull request fixes the bug in resnet_v1 and resnet_v2

### CODE REVIEW

**Comment by ahundt** [comment]
@stef716 @nealwu this pull request is relevant to you guys

**Comment by nealwu** [comment]
Looks good. Thanks!
